examples/server/a_bit_of_everything.go

```diff
@@ -248,6 +249,21 @@ func (s *_ABitOfEverythingServer) Timeout(ctx context.Context, msg *empty.Empty)
 	}
 }
 
+func (s *_ABitOfEverythingServer) ErrorWithDetails(ctx context.Context, msg *empty.Empty) (*empty.Empty, error) {
+	stat := status.New(codes.Unknown, "with details")
```

Comment:
I think it would be better to collapse the next few lines

```
stat, err := status.New(codes.Unknown, "with details").
  WithDetails([]proto.message{
  ...
  })
```